src/jvm/main/org/jetbrains/kotlinx/lincheck/strategy/managed/ManagedStrategy.kt

```diff
@@ -380,18 +395,23 @@ abstract class ManagedStrategy(
      * @param iThread the number of the executed thread according to the [scenario][ExecutionScenario].
      */
     open fun onFinish(iThread: Int) {
+        // onActorFinish we be called so the statement below is not necessary, but we verify
+        // additionally that we are now in a testing code to avoid bugs
```

Comment:
"but we verify additionally that we are now in a testing code to avoid bugs" - unclear